lib/src/main/java/net/ypresto/androidtranscoder/MediaTranscoder.java

```diff
@@ -210,6 +233,11 @@ public void run() {
         void onTranscodeCompleted();
 
         /**
+         * Called when transcode canceled.
+         */
+        void onTranscodeCanceled();
```

Comment:
`Future#get()` will raise CancellationException if canceled. So this method is not necessary.